### ISSUE TITLE
CSV Export with HTML description and short description and Import issue on that file

### ISSUE THREAD

**Issue by parmarhardip** [woocommerce/woocommerce#25832]
**Describe the bug**
A clear and concise description of what the bug is. Please be as descriptive as possible; issues lacking detail, or for any other reason than to report a bug, may be closed without action.

**To Reproduce**
1. Go to 'Product Export'
2. Click on 'Export all the data with description and short description HTML.'
3. Open the file in Excel (I am using ubuntu so I am checking on `LibreOffice Calc` app) 
4. Change the name of any product
5. Save this file and import in the product importer it.
6. See error [screenshot](https://i.imgur.com/VaDVWOo.png)

**Screenshots**

**Expected behavior**

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress [Storefront] theme active (https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by juliaamosova** [comment]
Hi @parmarhardip,

Thank you for submitting the issue. However, it is not complete in order for us to take any action on it. 

Few things:

>Click on 'Export all the data with description and short description HTML.'

We don't have that option in a built-in Product Exporter:

<img width="1435" alt="Screen Shot 2020-03-11 at 2 41 49 PM" src="https://user-images.githubusercontent.com/19143190/76452006-c17ae880-63a6-11ea-8994-c9f7e66b75df.png">

Are you using a plugin to export products?

>Open the file in Excel (I am using ubuntu so I am checking on LibreOffice Calc app)
Change the name of any product

Can you please provide the exact html that is in your product description and short description?

When an import file contains HTML the file type can be mis-detected by WordPress on some servers.. Please see here for more details: https://docs.woocommerce.com/document/product-csv-importer-exporter/#section-13 - could it be your case?

Basically, we need more details in order to fully reproduce this issue. Thanks!

**Comment by juliaamosova** [comment]
Hi @parmarhardip,

As a part of this repository’s maintenance, I am closing this issue due to inactivity. Please feel free to comment on it in case we missed something. We’d be happy to take another look.
